syntax = "proto3";

package TensorRTConfig;

message Config {
  repeated string input_tensor_names = 1;
  repeated string output_tensor_names = 2;
  string onnx_file = 3;
  string engine_file = 4;
  // Specify the DLA core to run network on.
  int32 dla_core = 5;
  // Number of inputs in a batch
  int32 batch_size = 6;
  // Allow runnning the network in Int8 mode.
  bool int8 = 7;
  // Allow running the network in FP16 mode.
  bool fp16 = 8;
  // Allow running the network in BF16 mode.
  bool bf16 = 9;
}